### DISCUSSION TITLE
Nether roof damage

### DISCUSSION THREAD

**Discussion by IceWaffles** [PurpurMC/Purpur#1114] [__Completed__] [answered]
Add configuration options for nether roof damage. E.g. damage per second and a message warning users to get off. Would be super cool as players die almost instantly while going up because they didn't read rules and got told to not go up :')

**Comment by encode42** [comment]
[One part already exists in Paper](https://docs.papermc.io/paper/reference/world-configuration#nether-ceiling-void-damage-height)

[The other part already exists in Purpur](https://purpurmc.org/docs/Configuration/#void-damage-dealt)

Paper's Nether roof damage patch deals void damage, so Purpur's option will apply to the Nether roof as well.

**Reply by IceWaffles** [reply]
Ok, but the message sent to players then?

**Reply by emielderckx** [reply]
Added in [PurpurExtras](https://github.com/PurpurMC/PurpurExtras/wiki#block-building-above-nether).

**Comment by encode42** [comment, marked as answer]
> Added in [PurpurExtras](https://github.com/PurpurMC/PurpurExtras/wiki#block-building-above-nether).
- @emielderckx['s message](https://github.com/PurpurMC/Purpur/discussions/1114#discussioncomment-3851747)
